### ISSUE TITLE
No thumbnails and can't upload images

### ISSUE THREAD

**Issue by mysteriomask** [grzegorznittner/chanu#476]
2 new issues just popped up within this last week.  When I view a board I don't get any thumbnails anymore or if I do I get 1 for every 10-15 threads.  When I click on a thread I can see thumbnails fine but when viewing the threads in a board I can't see any.  The other issue is I suddenly cannot reply with an image any longer.  Text is fine but when I click on the image one of 2 things happens. It will bring up the photo without showing any photos and immediately freeze the whole app, or the photo upload button just won't work at all.  

I'm using an HTC U11 Android 8.0.0.  The chanu I'm using is 2.0.22. Although I downloaded the 2.0.25 one the about in the app reads as. .22.   I have also tried clearing the cache.  Although I have not tried to uninstall chanu and reinstall yet.

**Comment by johnarleyburns** [comment]
Wow someone still uses it!

**Comment by SlayerOrnstein** [comment]
Yeah haven't found anything quiet as good

**Comment by SlayerOrnstein** [comment]
But yes can confirm this is a recent bug

**Comment by Greznor** [comment]
I'm looking into this now. Not all boards are telling to load thumbnails. Once I have that fixed, I'll need to figure out how to get it to build on the CI with the newer SDK. 

A lot of people all still use Chanu. Screenshots show up on /r/4chan.

**Comment by vidaluk0** [comment]
read on thread that it is because they changed the servers from is.4cdn.org to i.4cdn.org

**Comment by vidaluk0** [comment]
the image servers

**Comment by sulmanshah** [comment]
I use it as well it's my preferred 4chan app! I'm having the same issues with images loading. Anything I can do locally to make the application work? I didn't kode with klossy so I don't know how to CD.. CD..

**Comment by Greznor** [comment]
I fixed the thumbnail issue. I'll push a commit and merge shortly. The new version is 2.0.26

It's compiled using Android 27 (8.1) and the latest build tools.

**Comment by johnarleyburns** [comment]
Great, developing android these days?

**Comment by Greznor** [comment]
Not much anymore. Becoming a family man destroyed my free time.

**Comment by Greznor** [comment]
The pull request is ready.

**Comment by aerojun** [comment]
Nice

**Comment by HumbleJoe70** [comment]
Could somebody be kind enough to upload compiled apk , for us less tech savvy mortals.

**Comment by aerojun** [comment]
@HumbleJoe70 wait for the merge and then https://github.com/grzegorznittner/chanu/releases

**Comment by HumbleJoe70** [comment]
@aerojun That's the problem i cant even download older versions , all i see is app-debug.apk and source files , no normal apk ? em i not seeing a download button ?

**Comment by Greznor** [comment]
https://github.com/Greznor/chanu/releases/tag/v2.0.26

This is until the merge is finalized at the official repo.

Note: You may have to uninstall older versions of Chanu if the apk fails to install.

**Comment by vidaluk0** [comment]
Thank you so much Greznor, works just fine. Anyway I wonder why does the app need permission to contacts, location and modify system settings? 

I denied them anyway but find it odd

**Comment by mysteriomask** [comment]
That fixed the thumbnail problem.  The image uploader is still broke for me though.   Is that going to be worked on or is that a problem only I'm having and how can I go about fixing that.

**Comment by Greznor** [comment]
The permission thing is a weird bug. The manifest for the app only asks for storage, network, and phone state...

But while typing this, I think I figured out this bug.

**Comment by Greznor** [comment]
Is this all boards and any image? I just posted in /trash/ with a random image.

I'm on 7.1 with an m8. Monday I'll have my HTC 10 back. I can test to see if it's HTC's build of Android 8.0

**Comment by mysteriomask** [comment]
Seems to be all images but I think it's something on my end. The picture uploaded opens up files on my phone.  It's being very slow and freezes when I try to open file without going through Chanu so probably something on my end.

**Comment by Greznor** [comment]
Which gallery/file browser app are you using to select the image?

**Comment by vidaluk0** [comment]
Image upload work fine for me, just posted on /vg/. Using Android 8.0 Moto X4

**Comment by sulmanshah** [comment]
I'd like to try on my OnePlus 5t, how can I download the apk? F-droid has been giving me trouble lately too "error getting index index file" I was going to download logcat but I really don't feel like adb right now.

**Comment by mysteriomask** [comment]
When I select the button on chanu it sends me automatically to the app called files on my phone.  It automatically opens it to the downloads folder.  I have a feeling I have too much junk in it and need to move it.  I'm able to make it work in the app itself albeit slowly.  In chanu it just fully freezes.  Later I'll try to empty the folder out and see if that fixes anything.

**Comment by mysteriomask** [comment]
Cleaning out my download folder fixed my image uploading issue.  Although it was odd that it started happening the same time the thumbnail bug happened it was working perfectly fine before that.

**Comment by Greznor** [comment]
About how many images did you have before cleaning up? I'm probably not far behind you.

**Comment by mysteriomask** [comment]
It was a lot like 10-15k. I hadn't cleaned it out since I got the phone.  But besides image files it also included video files, stuff for work, documents, text other junk I randomly or accidentally downloaded.  Again the odd thing was I couldn't have downloaded that much new stuff since the the problem started for me so unless I just managed to download 1 file that was messing everything up or just hit the limit to start causing issues at the same time the thumbnail problem started then it was weirdly coincidental.  But it's working now.
